### ISSUE TITLE
[Wrong device]: LEDVANCE variant of different size 120x30 instead of 60x60.

### ISSUE THREAD

**Issue by LayerCakeMakes** [Koenkk/zigbee2mqtt#17547]
### Link

https://www.ledvance.de/consumer/produkte/smart-home/smart-leuchten/smart-innenraum-leuchten/smart-panel-leuchten-mit-wifi-technologie/panel-leuchten-mit-rahmen-und-wifi-technologie/wand--und-deckenleuchten-mit-zigbee-technologie-c6414

### Model

4058075181519

### Description

SMART+ panel 120 x 30cm tunable white

### Vendor

LEDVANCE

### Picture (link)

https://www.ledvance.de/05_LEDVANCE_and_Partner/1803722/image-thumb__1803722__product_detail/asset-1803722_Planon_Plus_30x120.webp

### Database entry

{"id":5,"type":"Router","ieeeAddr":"0xf0d1b8000013ce93","nwkAddr":12052,"manufId":4489,"manufName":"LEDVANCE","powerSource":"Mains (single phase)","modelId":"Panel TW Z3","epList":[1,242],"endpoints":{"1":{"profId":260,"epId":1,"devId":268,"inClusterList":[0,3,4,5,6,8,768,2821,4096,64527],"outClusterList":[25],"clusters":{"genBasic":{"attributes":{"modelId":"Panel TW Z3","manufacturerName":"LEDVANCE","powerSource":1,"zclVersion":2,"appVersion":1,"stackVersion":2,"hwVersion":1,"dateCode":"Sep 29 2021\u0000\u0000\u0000\u0000\u0000","swBuildId":"01056400"}},"lightingColorCtrl":{"attributes":{"colorCapabilities":16,"colorTempPhysicalMin":153,"colorTempPhysicalMax":370,"colorMode":2,"colorTemperature":153}},"genOnOff":{"attributes":{"onOff":1}},"genLevelCtrl":{"attributes":{"currentLevel":254}}},"binds":[],"configuredReportings":[],"meta":{}},"242":{"profId":41440,"epId":242,"devId":97,"inClusterList":[],"outClusterList":[33],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":1,"stackVersion":2,"hwVersion":1,"dateCode":"Sep 29 2021\u0000\u0000\u0000\u0000\u0000","swBuildId":"01056400","zclVersion":2,"interviewCompleted":true,"meta":{"configured":2},"lastSeen":1682982464589,"defaultSendRequestWhen":"immediate"}

### Notes

Model number according to the name of the product pdf which seems to have been used for the 60x60 version too.
Basically a completely different product but probably using the same electronics for both sizes.
